src/helm/benchmark/scenarios/lextreme_scenario.py

```diff
@@ -0,0 +1,276 @@
+            "I-ORGANIZACAO",
+            "B-PESSOA",
+            "I-PESSOA",
+            "B-TEMPO",
+            "I-TEMPO",
+            "B-LOCAL",
+            "I-LOCAL",
+            "B-LEGISLACAO",
+            "I-LEGISLACAO",
+            "B-JURISPRUDENCIA",
+            "I-JURISPRUDENCIA",
+        ],
+        "legalnero": [
+            'O',
+            'B-TIME',
+            'I-TIME',
+            'B-LEGAL',
+            'I-LEGAL',
+            'B-ORG',
+            'I-ORG',
+            'B-LOC',
+            'I-LOC',
+            'B-PER',
+            'I-PER',
+        ], "greek_legal_ner": [
+            'O',
+            'B-ORG',
+            'I-ORG',
+            'B-GPE',
+            'I-GPE',
+            'B-LEG-REFS',
+            'I-LEG-REFS',
+            'B-PUBLIC-DOCS',
+            'I-PUBLIC-DOCS',
+            'B-PERSON',
+            'I-PERSON',
+            'B-FACILITY',
+            'I-FACILITY',
+            'B-LOCATION-UNK',
+            'I-LOCATION-UNK',
+            'B-LOCATION-NAT',
+            'I-LOCATION-NAT'
+        ], "mapa_course": [
+            'O',
+            'B-ORGANISATION',
+            'I-ORGANISATION',
+            'B-ADDRESS',
+            'I-ADDRESS',
+            'B-DATE',
+            'I-DATE',
+            'B-PERSON',
+            'I-PERSON',
+            'B-AMOUNT',
+            'I-AMOUNT',
+            'B-TIME',
+            'I-TIME'
+        ], "mapa_fine": [
+            'O',
+            'B-BUILDING',
+            'I-BUILDING',
+            'B-CITY',
+            'I-CITY',
+            'B-COUNTRY',
+            'I-COUNTRY',
+            'B-PLACE',
+            'I-PLACE',
+            'B-TERRITORY',
+            'I-TERRITORY',
+            'I-UNIT',
+            'B-UNIT',
+            'B-VALUE',
+            'I-VALUE',
+            'B-YEAR',
+            'I-YEAR',
+            'B-STANDARD ABBREVIATION',
+            'I-STANDARD ABBREVIATION',
+            'B-MONTH',
+            'I-MONTH',
+            'B-DAY',
+            'I-DAY',
+            'B-AGE',
+            'I-AGE',
+            'B-ETHNIC CATEGORY',
+            'I-ETHNIC CATEGORY',
+            'B-FAMILY NAME',
+            'I-FAMILY NAME',
+            'B-INITIAL NAME',
+            'I-INITIAL NAME',
+            'B-MARITAL STATUS',
+            'I-MARITAL STATUS',
+            'B-PROFESSION',
+            'I-PROFESSION',
+            'B-ROLE',
+            'I-ROLE',
+            'B-NATIONALITY',
+            'I-NATIONALITY',
+            'B-TITLE',
+            'I-TITLE',
+            'B-URL',
+            'I-URL',
+            'B-TYPE',
+            'I-TYPE',
+        ],
+    }
+
+    def __init__(self, subset: str):
+        assert subset in task_code_mapping.keys(), f"Unknown subset: {subset}"
+        self.subset = subset
+
+    def get_instances_for_subset(self, config: str) -> List[Instance]:
+        task_code = task_code_mapping[config]
+        # Load dataset
+        dataset = load_dataset(self.dataset_name, config)
```

Comment:
I'd suggest putting setting `cache_dir` in [the same way as RAFT scenario](https://github.com/stanford-crfm/helm/blob/e2bcaa3e6297fa38106889a0dbf012dd85d62ec9/src/helm/benchmark/scenarios/raft_scenario.py#L106-L109), which will put the cache under the benchmark output, rather than in the user folder.